### PR TITLE
feat(replays): add migration for replay_id in discover

### PR DESCRIPTION
**Pull request by JoshFerge**
split out from https://github.com/getsentry/snuba/pull/5787

### CODE REVIEW

File: snuba/migrations/group_loader.py [hunk 1]
```diff
@@ -330,8 +331,6 @@ def get_migrations(self) -> Sequence[str]:
             "0037_add_record_meta_column_sets",
             "0038_add_record_meta_column_distributions",
             "0039_add_record_meta_column_gauges",
-            "0040_remove_counters_meta_tables",
-            "0041_adjust_partitioning_meta_tables",
```
Comment (cmanallen) [inline snuba/migrations/group_loader.py:334]: Why were these removed?

**Review by cmanallen — CHANGES_REQUESTED** [review]

File: snuba/migrations/group_loader.py (hunk 1 above)
Comment (cmanallen) [reply snuba/migrations/group_loader.py]: Oh lol my review was too slow.

Comment (JoshFerge) [reply snuba/migrations/group_loader.py]: whoops, fixed. merge issue

File: snuba/migrations/group_loader.py [hunk 2]
```diff
@@ -330,8 +331,6 @@ def get_migrations(self) -> Sequence[str]:
             "0037_add_record_meta_column_sets",
             "0038_add_record_meta_column_distributions",
             "0039_add_record_meta_column_gauges",
-            "0040_remove_counters_meta_tables",
```
[the remark this answers is not in the document]
Comment (JoshFerge) [reply snuba/migrations/group_loader.py]: fixed, sorry about that.

**Review by cmanallen — APPROVED** [review]
lgtm

**Review by evanh — APPROVED** [review]
My understanding is that this should not be merged until _after_ https://github.com/getsentry/snuba/pull/5788.

So I am approving, but please wait until that other PR is merged.

**Comment by JoshFerge** [comment]
am i good to merge this?

**Pull request merged**